web-wallet/src/lib/components/Allocate/Allocate.svelte

```diff
@@ -117,21 +168,15 @@
           <div class="operation__input-wrapper">
             <Textbox
               className="operation__input-field"
-              bind:value={shieldedAmount}
+              value={shieldedNumber}
               required
               type="number"
               min={minAmount}
-              max={deductLuxFeeFrom(
-                luxToDusk(shieldedBalance + unshieldedBalance),
-                fee
-              )}
+              max={difference > 0n
```

Comment:
Maybe everything could be more readable by adding a reactive boolean variable instead of checking constantly for `difference > 0n` and `difference < 0n`?